libraries/chain/include/eosio/chain/transaction.hpp

```diff
@@ -126,14 +126,9 @@ namespace eosio { namespace chain {
       signed_transaction( signed_transaction&& ) = default;
       signed_transaction& operator=(const signed_transaction&) = delete;
       signed_transaction& operator=(signed_transaction&&) = default;
-      signed_transaction( transaction&& trx, const vector<signature_type>& signatures, const vector<bytes>& context_free_data)
+      signed_transaction( transaction&& trx, vector<signature_type> signatures, vector<bytes> context_free_data)
```

Comment:
You could also do `transaction` instead of `transaction&&` for more generality.

Reply:
Done